index.bs

```diff
@@ -630,6 +630,33 @@ spec:url; type:dfn; text:scheme
     portals-cross-origin-load.sub.html
   </wpt>
 
+  <section algorithm="htmlportalelement-activation-behavior">
+    A <{portal}> element's [=activation behavior=] is to run the following steps:
+
+    1. Let |portalBrowsingContext| be the [=guest browsing context=] of [=this=].
+
+    1. If |portalBrowsingContext| is null, abort these steps.
+
+    1. Let |predecessorBrowsingContext| be the [=document browsing context|browsing context=] of
+        [=this=]'s [=node document=].
+
+    1. If |predecessorBrowsingContext| is null, abort these steps.
+
+    1. If the [=portal state=] of |predecessorBrowsingContext| is not "`none`", abort these steps.
+
+    1. Let |sourceOrigin| be [=this=]'s [=relevant settings object=]'s
+        [=environment settings object/origin=].
+
+    1. Run the steps to [=activate a portal browsing context|activate=] |portalBrowsingContext|
+        in place of |predecessorBrowsingContext| with |sourceOrigin|.
+
+    <div class="note">
+      This is substantially similar to the steps in {{activate(options)}}, with default options.
+      User agents may wish to display suitable console messages under the same conditions that would
```

Comment:
```suggestion
      User agents might wish to display suitable console messages under the same conditions that would
```

Ideally don't use may/must/should in notes.

Reply:
Sure, okay. I don't think there's much risk of it being misinterpreted as normative, but "might wish" works fine too. :)